dataprofiler/profilers/json_decoder.py

```diff
@@ -0,0 +1,65 @@
+"""Contains methods to decode components of a Profiler."""
+
+import json
+
+from .base_column_profilers import BaseColumnProfiler
+from .categorical_column_profile import CategoricalColumn
+
+
+def get_column_profiler_class(class_name: str) -> BaseColumnProfiler:
+    """
+    Use name of class to return default-constructed version of that class.
+
+    Raises ValueError if class_name is not name of a subclass of
+        BaseColumnProfiler.
+
+    :param class_name: name of BaseColumnProfiler subclass retreived by
+        calling type(instance).__name__
+    :type class_name: str representing name of class
+    :return: subclass of BaseColumnProfiler object
+    """
+    if class_name == CategoricalColumn.__name__:
```

Comment:
I see, we are being declarative here to allow any class to load.

Reply:
also avoids circular imports since not in the base.

Reply:
I think to be optimum for future code we can use a dict:
```python
profiles = {
    CategoricalColumn.__name__: CategoricalColumn
}
if class_name not in profiles:
    raise V...
return profiles.get(class_name)(None)
```

Reply:
Sounds good, I'll make this change